### DISCUSSION TITLE
Strongswan  compiled version for windows exits directly

### DISCUSSION THREAD

**Discussion by yft22** [strongswan/strongswan#768] [Q&A]
Hello everyone,

On  a Lubuntu, I have cross compiled strongswan sources for a windows 64 bits target following the instructions from https://wiki.strongswan.org/projects/strongswan/wiki/windows

Everything went fine, the following files were generated:
- charon-svc.exe
- ipsecdump.exe
- swanctl.exe
- .libs directory with libstrongswan-0.dll

When I execute in a command line  charon-svc.exe or swanctl.exe, the programs exits directly with no log even when I launch the following commands:
-charon-svc.exe --help
-swanctl.exe --help

Am I missing something?

**Comment by tobiasbrunner** [comment]
No idea, but I'd try putting the .dll files (there should be more than just libstrongswan, did you use `make install` with `DESTDIR` as suggested on the wiki page?) in the same directory as the executables. Also, there is no `--help` for `charon-svc`, see [charon-svc](https://wiki.strongswan.org/projects/strongswan/wiki/Charon-svc) for details.

**Reply by yft22** [reply]
You are right I have copied some files manually but some was missing. Thanks!
